xml-csharp/SelectingXmlNodesWithXpath/SelectingXmlNodesWithXpathTests/SelectingXmlNodesWithXpathTest.cs

```diff
@@ -0,0 +1,123 @@
+using System.Diagnostics;
+using System.Xml;
+using System.Xml.Linq;
+using Xunit;
+
+namespace SelectingXmlNodesWithXpathTests
+{
+    public class SelectingXmlNodesWithXpathTest
+    {
+        XmlDocument Document { get; set; }
+        Dictionary<string, string> ExpectedResults { get; set; }
+
+
+        public SelectingXmlNodesWithXpathTest()
+        {
+            try
+            {
+                Document = new XmlDocument();
+                Document.Load("BooksCatalog.xml");
+
+                ExpectedResults = new Dictionary<string, string>()
+                {
+                    {
+                        "Book1",
+                        "<book id=\"1\">" +
+                            "<author>King, Stephen</author>" +
+                            "<title>IT</title>" +
+                            "<genre>Horror</genre>" +
+                            "<price>40.00</price>" +
+                        "</book>"
+                    },
+                    {
+                        "Book2",
+                        "<book id=\"2\">" +
+                            "<author>Assis, Machado De</author>" +
+                            "<title>Dom Casmurro</title>" +
+                            "<genre>Romance</genre>" +
+                            "<price>50.00</price>" +
+                        "</book>"
+                    },
+                    {
+                        "Book3",
+                        "<book id=\"3\">" +
+                            "<author>Calaprice, Alice; Lipscombe, Trevor</author>" +
+                            "<title>Albert Einstein: A Biography</title>" +
+                            "<genre>Biography</genre>" +
+                            "<price>30.00</price>" +
+                        "</book>"
+                    },
+                    {
+                        "Book4",
+                        "<book id=\"4\" xmlns=\"urn:example-schema\">" +
+                            "<author>Fowler, Martin; Beck, Kent</author>" +
+                            "<title>Refactoring: Improving the design of existing code</title>" +
+                            "<genre>Scientific</genre>" +
+                            "<price>60.00</price>" +
+                        "</book>"

```

Comment:
We can use raw string literals here to make this code a little nicer and easier to maintain.